### ISSUE TITLE
Consider pushing Z3 debug builds to Ivy

### ISSUE THREAD

**Issue by cheshire** [sosy-lab/java-smt#84]
@PhilippWendler Resolving issues with Z3 is much easier with a debug build handy, and it takes a while to produce.
Do you think it might be worth it to push it to Ivy, under used configuration, and then  it would be a matter of changing the configuration in `ivy.xml` in order to use it?
The downside is that the resulting shared object is ~160M.

**Comment by PhilippWendler** [comment]
How often do we need such a debug build? I would suspect on average less than once for each version of Z3 that we have published, so maybe it wouldn't even pay off. When I need a debug build, I use the latest git revision of Z3 anyway, not the latest version we had published.

Given that we currently have 17 published versions of Z3 in the repository, if we had adopted this idea from the beginning on we would now have already almost 3 GB of debug binaries.

So while the idea is good, it seems to not be worth it.
